### PR TITLE
Add example on how to use `SessionMiddleware`

### PR DESCRIPTION
**Pull request by Bernardoow**
# Summary

Add example to how to use session middleware. I made this PR because I found the documentation but the path to middleware is not present. So I had to search on code.

# Checklist

- [X] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
- [X] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [X] I've updated the documentation accordingly.

### CODE REVIEW

File: docs/middleware.md [hunk 1]
```diff
@@ -107,6 +107,20 @@ The following arguments are supported:
...
 
+```python
+from starlette.applications import Starlette
+from starlette.middleware import Middleware
+from starlette.middleware.sessions import SessionMiddleware
+
+routes = ...
+
+middleware = [
+    Middleware(SessionMiddleware, https_only=True)
```
Comment (aminalaee) [inline docs/middleware.md]: I think you need the `secret_key` argument for `SessionMiddleware`

Comment (Bernardoow) [reply docs/middleware.md]: @aminalaee you're right.

image

I'm going to fix it now.. sorry. !!

File: docs/middleware.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ from starlette.middleware.sessions import SessionMiddleware
 routes = ...
 
 middleware = [
-    Middleware(SessionMiddleware, https_only=True)
+    Middleware(SessionMiddleware, secret_key="R4ND0M_S3CR3T_K3Y", https_only=True)
```
Comment (Kludex) [inline docs/middleware.md]:
```suggestion
    Middleware(SessionMiddleware, secret_key=..., https_only=True)
```

Comment (Bernardoow) [reply docs/middleware.md]: @Kludex done here 8e1128d

**Review by Kludex — APPROVED** [review]

**Pull request merged**